src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -71,180 +71,151 @@ const defaultProps = {
-                                accessibilityLabel={this.props.translate('workspace.inviteMessage.personalMessagePrompt')}
-                                autoCompleteType="off"
-                                autoCorrect={false}
-                                autoGrowHeight
-                                containerStyles={[this.props.themeStyles.autoGrowHeightMultilineInput]}
-                                defaultValue={this.state.welcomeNote}
-                                value={this.state.welcomeNote}
-                                onChangeText={(text) => {
-                                    this.debouncedSaveDraf(text);
-                                    this.setState({welcomeNote: text});
-                                }}
-                            />
-                        </View>
-                    </Form>
-                </FullPageNotFoundView>
-            </ScreenWrapper>
-        );
-    }
+                    <View style={[props.themeStyles.mv4, props.themeStyles.justifyContentCenter, props.themeStyles.alignItemsCenter]}>
+                        <MultipleAvatars
+                            size={CONST.AVATAR_SIZE.LARGE}
+                            icons={OptionsListUtils.getAvatarsForAccountIDs(_.values(props.invitedEmailsToAccountIDsDraft), props.allPersonalDetails, props.invitedEmailsToAccountIDsDraft)}
+                            shouldStackHorizontally
+                            shouldDisplayAvatarsInRows
+                            secondAvatarStyle={[props.themeStyles.secondAvatarInline]}
+                        />
+                    </View>
+                    <View style={[props.themeStyles.mb5]}>
+                        <Text>{props.translate('workspace.inviteMessage.inviteMessagePrompt')}</Text>
+                    </View>
+                    <View style={[props.themeStyles.mb3]}>
+                        <InputWrapper
+                            InputComponent={TextInput}
+                            role={CONST.ACCESSIBILITY_ROLE.TEXT}
+                            inputID="welcomeMessage"
+                            label={props.translate('workspace.inviteMessage.personalMessagePrompt')}
+                            accessibilityLabel={props.translate('workspace.inviteMessage.personalMessagePrompt')}
+                            autoCompleteType="off"
+                            autoCorrect={false}
+                            autoGrowHeight
+                            inputStyle={[props.themeStyles.verticalAlignTop]}
+                            containerStyles={[props.themeStyles.autoGrowHeightMultilineInput]}
+                            value={welcomeMessage}
+                            onChangeText={(text) => {
+                                saveDraft(text);
+                            }}
+                            ref={(el) => (welcomeMessageInputRef.current = el)}
```

Comment:
```suggestion
                            ref={welcomeMessageInputRef}
```

Reply:
Won't be changing that due to using `useAutoFocusInput` and having to update the multiline input range before assigning ref.